openapi/python.sh

```diff
@@ -54,10 +54,10 @@ kubeclient::generator::generate_client "${OUTPUT_DIR}"
 echo "--- Patching generated code..."
 
 # workaround https://github.com/swagger-api/swagger-codegen/pull/8401
-find "${OUTPUT_DIR}/client/" -type f -name \*.py -exec sed -i 's/async=/async_req=/g' {} +
-find "${OUTPUT_DIR}/client/" -type f -name \*.py -exec sed -i 's/async bool/async_req bool/g' {} +
-find "${OUTPUT_DIR}/client/" -type f -name \*.py -exec sed -i "s/'async'/'async_req'/g" {} +
-sed -i "s/if not async/if not async_req/g" "${OUTPUT_DIR}/client/api_client.py"
+find "${OUTPUT_DIR}/" -type f -name \*.py -exec sed -i 's/async=/async_req=/g' {} +
```

Comment:
It can replace too much. I suggest replacing all occurrences of `client` with `$PACKAGE_NAME` (there are other patches, `CLEANUP_DIRS` etc.).

Reply:
@tomplus thanks for the clarification. I need to to withdraw this RP and re-submit a new one to clear the CLA block. Will take your recommendation to replace client with $PACKAGE_NAME  thanks.